Scheduler: add error descriptions (#5807)
DevExpress/devextreme-documentation: api-reference/10 UI Components/Errors and Warnings/E1058.md

---
---
##### shortDescription
Occurs in [Scheduler](/api-reference/10%20UI%20Components/dxScheduler '/Documentation/ApiReference/UI_Components/dxScheduler/') when [startDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/startDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#startDayHour') is set to a time that is later than [endDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/endDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#endDayHour') or at least one of these property values is not an integer.

---

> To remove this error, check that both properties are integers in the [0, 24] range and specify a **startDayHour** that is earlier than the **endDayHour**.

Reviewer: ```suggestion
To remove this error, check that both properties are integers in the [0, 24] range and specify a **startDayHour** before the **endDayHour**.
```
